packages/airnode-protocol/src/index.ts

```diff
@@ -17,43 +17,13 @@ import {
   RequesterAuthorizerWithAirnode__factory as RequesterAuthorizerWithAirnodeFactory,
   RrpBeaconServer__factory as RrpBeaconServerFactory,
 } from './contracts';
-import AirnodeRrpDeploymentMainnet from '../deployments/mainnet/AirnodeRrp.json';
-import AccessControlRegistryDeploymentMainnet from '../deployments/mainnet/AccessControlRegistry.json';
-import RequesterAuthorizerWithAirnodeDeploymentMainnet from '../deployments/mainnet/RequesterAuthorizerWithAirnode.json';
-import AirnodeRrpDeploymentRopsten from '../deployments/ropsten/AirnodeRrp.json';
-import AccessControlRegistryDeploymentRopsten from '../deployments/ropsten/AccessControlRegistry.json';
-import RequesterAuthorizerWithAirnodeDeploymentRopsten from '../deployments/ropsten/RequesterAuthorizerWithAirnode.json';
-import AirnodeRrpDeploymentRinkeby from '../deployments/rinkeby/AirnodeRrp.json';
-import AccessControlRegistryDeploymentRinkeby from '../deployments/rinkeby/AccessControlRegistry.json';
-import RequesterAuthorizerWithAirnodeDeploymentRinkeby from '../deployments/rinkeby/RequesterAuthorizerWithAirnode.json';
-import AirnodeRrpDeploymentGoerli from '../deployments/goerli/AirnodeRrp.json';
-import AccessControlRegistryDeploymentGoerli from '../deployments/goerli/AccessControlRegistry.json';
-import RequesterAuthorizerWithAirnodeDeploymentGoerli from '../deployments/goerli/RequesterAuthorizerWithAirnode.json';
-import AirnodeRrpDeploymentKovan from '../deployments/kovan/AirnodeRrp.json';
-import AccessControlRegistryDeploymentKovan from '../deployments/kovan/AccessControlRegistry.json';
-import RequesterAuthorizerWithAirnodeDeploymentKovan from '../deployments/kovan/RequesterAuthorizerWithAirnode.json';
+import references from '../deployments/references.json';
+
+const AirnodeRrpAddresses: { [chainId: number]: string } = references.AirnodeRrp;
+const AccessControlRegistryAddresses: { [chainId: number]: string } = references.AccessControlRegistry;
```

Comment:
I'm not sure that the explicit typing is really needed here but it's fine to leave it in

Reply:
I mostly did it because I couldn't figure out how to do it without the typing :man_facepalming: